apps/meteor/client/views/setupWizard/steps/RegisterServerStep.tsx

```diff
@@ -48,6 +49,7 @@ const RegisterServerStep = (): ReactElement => {
 			onSuccess: () => {
 				queryClient.invalidateQueries(['licenses']);
 				completeSetupWizard();
+				homeRoute.push();
```

Comment:
why not add this redirect inside the `completeSetupWizard` ?

Reply:
The redirect is already implemented in another place. Something is not triggering the re-render effect...

Reply:
Upon looking more into it, the re-direct is already implemented in a hook called `useRouteLock` the issue is, this hook his not redirecting because the value of setting `Show_Setup_Wizard` is not changing upon completion of setup wizard due to TOTP and totp modal doesn't appear on setup-wizard. Looking more into this, if we can show totp modal here or skip the totp check 🤔

Reply:
After talking to Tasso, we decided to implement a new meteor method which won't be guarded by totp, this method can be used to set some of the setup wizard settings.